xbmc/video/dialogs/GUIDialogVideoInfo.h

```diff
@@ -107,11 +107,11 @@ class CGUIDialogVideoInfo :
   static bool OnGetFanart(const CFileItemPtr &videoItem);
 
   CFileItemPtr m_movieItem;
-  CFileItemList *m_castList;
-  bool m_bViewReview;
-  bool m_bRefresh;
-  bool m_bRefreshAll;
-  bool m_hasUpdatedThumb;
-  bool m_hasUpdatedUserrating;
-  int m_startUserrating;
+  CFileItemList *m_castList = new CFileItemList;
+  bool m_bViewReview = false;
+  bool m_bRefresh = false;
+  bool m_bRefreshAll = true;
+  bool m_hasUpdatedThumb = false;
+  bool m_hasUpdatedUserrating = false;
+  int m_startUserrating = -1;
```

Comment:
If there is only one constructor for the class I see no benefit in initializing the members in the header file. The code gets imo not more readable or more maintainable - there is no difference compared to init in the cpp file.

But there is one drawback: If you change an init value, all cpp files which include that header file must be recompiled if the init code is in the header file. If you have the init code in the cpp file, only the cpp file must be recompiled if you change an init value.